docs/api/non_geo_datasets.csv

```diff
@@ -1,34 +1,34 @@
 Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
-`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,186x1,122",0.5,MSI
+`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,122x1,186",10,MSI
```

Comment:
If you run `gdalinfo` on the files in @calebrob6's VM, the pixel size is 10, not 0.5. Also, dimensions are reversed.